app/views/jobs/show.html.erb

```diff
@@ -1,13 +1,64 @@
-Title: <%= @job.title %>
-<br>
+<table class="index-display">
+  <tr>
+    <th>Title</th>
+    <td><%= @job.title %></td>
+    <td>
+      <div class="edit-item">
+        <%= link_to "Edit", edit_company_job_path(@job.company, @job) %>
+      </div>
+    </td>
+  </tr>
+  <tr>
+    <th>Description</th>
+    <td><%= @job.description %></td>
+    <td>
+      <div class="delete-item">
+        <%= link_to "Delete", company_job_path(@job.company, @job), method: :delete %>
+      </div>
+    </td>
+  </tr>
+  <tr>
+    <th>Level of Interest</th>
+    <td><%= @job.level_of_interest %></td>
+  </tr>
+  <tr>
+    <th>City</th>
+    <td><%= @job.city %></td>
+  </tr>
+  <tr>
+    <th>Company</th>
+    <td><%= link_to @job.company.name, company_path(@job.company) %></td>
+  </tr>
+</table>
 
-Description: <%= @job.description %>
-<br>
+<h2>Comments</h2>
+<hr>
 
-Level of Interest: <%= @job.level_of_interest %>
-<br>
+<% @job.comments.order(created_at: :desc).each do |comment| %>
```

Comment:
i suggest moving this query out of the view and into maybe a model method on Comment